packages/mgt-components/src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -99,20 +102,24 @@ $person-card-nav-back-arrow-hover-color: var(--person-card-nav-back-arrow-hover-
       --presence-background-color: #{$background-color};
       --presence-icon-color: #{$background-color};
       --avatar-size: 75px;
-      --font-size: #{$person-card-display-name-font-size};
+      --line1-font-size: #{$person-card-display-name-font-size};
+      --line1-line-height: #{$person-card-display-name-line-height};
       --color: #{$person-card-display-name-color};
       --line2-font-size: #{$person-card-title-font-size};
+      --line2-line-height: #{$person-card-title-line-height};
       --line2-color: #{$person-card-title-color};
       --line3-font-size: #{$person-card-subtitle-font-size};
+      --line3-line-height: #{$person-card-subtitle-line-height};
       --line3-color: #{$person-card-subtitle-color};
+      --details-cta-spacing: 30px;
     }
 
     .base-icons {
```

Comment:
This needs to have a z-index added so that the mgt-person doesn't sit overtop of this element where they overlap and prevent the mouse tracking for hover states.